rails_event_store/lib/rails_event_store/all.rb

```diff
@@ -22,8 +15,17 @@ module RailsEventStore
   InvalidPageStart            = RubyEventStore::InvalidPageStart
   InvalidPageSize             = RubyEventStore::InvalidPageSize
   CorrelatedCommands          = RubyEventStore::CorrelatedCommands
+  CorrelatedDecorator         = RubyEventStore::CorrelatedDecorator
```

Comment:
I am not a fan of this name `CorrelatedDecorator`. The fact that it is a dectorator is not interesting.